app/script.js

```diff
@@ -0,0 +1,41 @@
+$(document).ready(function() {
+	$("button").on("click", function() {
+		var url = $(this).attr("id");
+		$.get(url, function(response) {
+			var resText;
+			if (typeof response === "object" && url == "jokes") {
+				resText = response.setup + ": " + response.punchline;
+			} else if (typeof response === "object" && url == "characters") {
+				resText = response.character + " is a character from " + response.story;
+			} else {
+				resText = response;
+			}
+			$("#ajax-text").text(resText);
+		});
+	});
+
+	$.get("characters", function(response) {
+		var resText = response.character + " from the book " + response.story;
+		$("#daily-character").text(resText);
+	});
```

Comment:
Are you using a setTimeout somewhere to make this a daily thing? If not, you could put this whole $.get in a setTimeout that ran each day.